desc/compute/_field.py

```diff
@@ -106,6 +106,220 @@ def _B_sup_zeta(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="B^phi",
+    label="B^{\\phi}",
+    units="T \\cdot m^{-1}",
+    units_long="Tesla / meter",
+    description="Contravariant cylindrical toroidal angle component of magnetic field",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["B0", "phi_z", "lambda_t", "phi_t", "lambda_z"],
+)
+def _B_sup_phi(params, transforms, profiles, data, **kwargs):
+    data["B^phi"] = data["B0"] * (
```

Comment:
This is written in this form, independent of iota, so that it can be computed without performing any integrations.